lms/static/js/student_account/views/LoginView.js

```diff
@@ -126,7 +126,9 @@
                                 paragraphEnd: HtmlUtils.HTML('</p>'),
                                 email: email,
                                 platform_name: this.platformName,
-                                anchorStart: HtmlUtils.HTML('<a href="' + this.supportURL + '">'),
+                                anchorStart: HtmlUtils.HTML(
+                                    '<a target="_blank" href="' + this.passwordResetSupportURL + '">'
```

Comment:
Ah, I based it off my own intuition of user experience (I hate losing my current window). But I'll leave it since no request was made.